modules/cron2/Classes/Cron2.php

```diff
@@ -4,48 +4,109 @@
 
 class Cron2
 {
+
+    const LS_CRON_MAX_ERRORS = 3; //Maximum amount of errors for a job
```

Comment:
We don't need to prefix the constant with `LS_`, because this constant only exists in the scope of the `Cron2` class.

I suggest to rename this constant to 
```suggestion
    /**
     * Maximum amount of errors for a job
     */
    const CRON_MAX_ERRORS = 3;
```
and using PHP doc